pkgs/development/interpreters/php/fix-tokenizer-php81.patch

```diff
@@ -0,0 +1,11 @@
+diff --git a/ext/tokenizer/Makefile.frag b/ext/tokenizer/Makefile.frag
+index 745cbfce91..7970027607 100644
+--- a/ext/tokenizer/Makefile.frag
++++ b/ext/tokenizer/Makefile.frag
+@@ -1,5 +1,4 @@
+-$(top_srcdir)/ext/tokenizer/tokenizer_data.c: $(top_srcdir)/Zend/zend_language_parser.y
++$(top_srcdir)/ext/tokenizer/tokenizer_data.c:
```

Comment:
Here not updating the variable essentially removes the rule, which can lead to the data being out of date.

Reply:
@stephank Updated its patch, I cherry picked it once again, could you verify now?